snuba/datasets/discover.py

```diff
@@ -343,7 +343,7 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             ApdexProcessor(),
             ImpactProcessor(),
             FailureRateProcessor(),
-            TimeSeriesColumnProcessor({}),
+            TimeSeriesColumnProcessor({"time": "timestamp"}),
```

Comment:
I did not add `rtime` which is a specific concept of the events dataset. 
Since bucketed time columns are creating virtual columns that are not declared in the dataset schema, I would add only the one that we have evidence of being used to not encourage this behavior (adding virtual columns out of the schema, specifically in this case where it is a column only present in events and not transactions).

Reply:
How does this processor work exactly? I thought the value is supposed to match the physical column name i.e. "finish_ts" in transactions and "timestamp" in events. Could we have a test for this case?

Reply:
It works this way.
This is a dataset processor. It is applied pre-translation. So this transforms 
```
time -> toStartoOfHour(timestamp)
```
then the translator transforms that into 
```
toStartoOfHour(timestamp) -> toStartoOfHour(timestamp) # if we picked events
toStartoOfHour(timestamp) -> toStartoOfHour(finish_ts) # if we pickjed transactions
```

The test is this one:
https://github.com/getsentry/snuba/blob/master/tests/test_discover_api.py#L454-L484

and it is not passing now on the AST. It passes after this PR.